From c82c83abefa4c6365e90a244fee959af9cb2d84d Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Wed, 30 Aug 2017 19:05:32 +0800
Subject: [PATCH 2501/5242] MLK-16327-2: ARM64: dts: fsl-imx8qm/qxp: correct
 edma channel name

commit  efbc9aee7846ddec8c56ef68c42144c21b5b3942 from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct edma channel name to support unique edma channel name in multi
edma instances.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi  |   34 +++++++++++-----------
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |   36 ++++++++++++------------
 2 files changed, 35 insertions(+), 35 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 52af095..ce4c827 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1852,11 +1852,11 @@
 			     <GIC_SPI 441 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 442 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 443 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-names = "edma-chan12-tx", "edma-chan13-tx",
-				  "edma-chan14-tx", "edma-chan15-tx",
-				  "edma-chan16-tx", "edma-chan17-tx",
-				  "edma-chan18-tx", "edma-chan19-tx",
-				  "edma-chan20-tx", "edma-chan21-tx";
+		interrupt-names = "edma0-chan12-rx", "edma0-chan13-tx",
+				  "edma0-chan14-rx", "edma0-chan15-tx",
+				  "edma0-chan16-rx", "edma0-chan17-tx",
+				  "edma0-chan18-rx", "edma0-chan19-tx",
+				  "edma0-chan20-rx", "edma0-chan21-tx";
 		status = "okay";
 	};
 
@@ -1893,13 +1893,13 @@
 				<GIC_SPI 315 IRQ_TYPE_LEVEL_HIGH>,
 				<GIC_SPI 317 IRQ_TYPE_LEVEL_HIGH>, /* sai1 */
 				<GIC_SPI 317 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-names = "edma-chan0-tx", "edma-chan1-tx", /* asrc0 */
-				"edma-chan2-tx", "edma-chan3-tx",
-				"edma-chan4-tx", "edma-chan5-tx",
-				"edma-chan6-tx", "edma-chan7-tx", /* esai0 */
-				"edma-chan8-tx", "edma-chan9-tx", /* spdif0 */
-				"edma-chan12-tx", "edma-chan13-tx", /* sai0 */
-				"edma-chan14-tx", "edma-chan15-tx"; /* sai1 */
+		interrupt-names = "edma2-chan0-rx", "edma2-chan1-rx", /* asrc0 */
+				"edma2-chan2-rx", "edma2-chan3-tx",
+				"edma2-chan4-tx", "edma2-chan5-tx",
+				"edma2-chan6-rx", "edma2-chan7-tx", /* esai0 */
+				"edma2-chan8-rx", "edma2-chan9-tx", /* spdif0 */
+				"edma2-chan12-rx", "edma2-chan13-tx", /* sai0 */
+				"edma2-chan14-rx", "edma2-chan15-tx"; /* sai1 */
 		status = "okay";
 	};
 
@@ -1926,11 +1926,11 @@
 				<GIC_SPI 330 IRQ_TYPE_LEVEL_HIGH>, /* sai6 */
 				<GIC_SPI 330 IRQ_TYPE_LEVEL_HIGH>,
 				<GIC_SPI 332 IRQ_TYPE_LEVEL_HIGH>; /* sai7 */
-		interrupt-names = "edma-chan0-tx", "edma-chan1-tx", /* asrc1 */
-				"edma-chan2-tx", "edma-chan3-tx",
-				"edma-chan4-tx", "edma-chan5-tx",
-				"edma-chan8-tx", "edma-chan9-tx", /* sai6 */
-				"edma-chan10-tx";                 /* sai7 */
+		interrupt-names = "edma3-chan0-rx", "edma3-chan1-rx", /* asrc1 */
+				"edma3-chan2-rx", "edma3-chan3-tx",
+				"edma3-chan4-tx", "edma3-chan5-tx",
+				"edma3-chan8-rx", "edma3-chan9-tx", /* sai6 */
+				"edma3-chan10-tx";                 /* sai7 */
 		status = "okay";
 	};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 457301d..f6c4f3f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1598,10 +1598,10 @@
 			     <GIC_SPI 439 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 440 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 441 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-names = "edma-chan8-tx", "edma-chan9-tx",
-				  "edma-chan10-tx", "edma-chan11-tx",
-				  "edma-chan12-tx", "edma-chan13-tx",
-				  "edma-chan14-tx", "edma-chan15-tx";
+		interrupt-names = "edma0-chan8-rx", "edma0-chan9-tx",
+				  "edma0-chan10-rx", "edma0-chan11-tx",
+				  "edma0-chan12-rx", "edma0-chan13-tx",
+				  "edma0-chan14-rx", "edma0-chan15-tx";
 		status = "okay";
 	};
 
@@ -1642,15 +1642,15 @@
 				<GIC_SPI 317 IRQ_TYPE_LEVEL_HIGH>,
 				<GIC_SPI 368 IRQ_TYPE_LEVEL_HIGH>,
 				<GIC_SPI 368 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-names = "edma-chan0-tx", "edma-chan1-tx", /* asrc0 */
-				"edma-chan2-tx", "edma-chan3-tx",
-				"edma-chan4-tx", "edma-chan5-tx",
-				"edma-chan6-tx", "edma-chan7-tx", /* esai0 */
-				"edma-chan8-tx", "edma-chan9-tx", /* spdif0 */
-				"edma-chan12-tx", "edma-chan13-tx", /* sai0 */
-				"edma-chan14-tx", "edma-chan15-tx", /* sai1 */
-				"edma-chan21-tx",		/* gpt5 */
-				"edma-chan23-tx";		/* gpt7 */
+		interrupt-names = "edma2-chan0-rx", "edma2-chan1-rx", /* asrc0 */
+				"edma2-chan2-rx", "edma2-chan3-tx",
+				"edma2-chan4-tx", "edma2-chan5-tx",
+				"edma2-chan6-rx", "edma2-chan7-tx", /* esai0 */
+				"edma2-chan8-rx", "edma2-chan9-tx", /* spdif0 */
+				"edma2-chan12-rx", "edma2-chan13-tx", /* sai0 */
+				"edma2-chan14-rx", "edma2-chan15-tx", /* sai1 */
+				"edma2-chan21-tx",		/* gpt5 */
+				"edma2-chan23-rx";		/* gpt7 */
 		status = "okay";
 	};
 
@@ -1677,11 +1677,11 @@
 				<GIC_SPI 330 IRQ_TYPE_LEVEL_HIGH>, /* sai4 */
 				<GIC_SPI 330 IRQ_TYPE_LEVEL_HIGH>,
 				<GIC_SPI 332 IRQ_TYPE_LEVEL_HIGH>; /* sai5 */
-		interrupt-names = "edma-chan0-tx", "edma-chan1-tx", /* asrc1 */
-				"edma-chan2-tx", "edma-chan3-tx",
-				"edma-chan4-tx", "edma-chan5-tx",
-				"edma-chan8-tx", "edma-chan9-tx", /* sai4 */
-				"edma-chan10-tx";                 /* sai5 */
+		interrupt-names = "edma3-chan0-rx", "edma3-chan1-rx", /* asrc1 */
+				"edma3-chan2-rx", "edma3-chan3-tx",
+				"edma3-chan4-tx", "edma3-chan5-tx",
+				"edma3-chan8-rx", "edma3-chan9-tx", /* sai4 */
+				"edma3-chan10-tx";                 /* sai5 */
 		status = "okay";
 	};
 
-- 
1.7.9.5

